dm/pkg/utils/db.go

```diff
@@ -632,3 +632,31 @@ func GetTableCreateSQL(ctx context.Context, conn *sql.Conn, tableID string) (sql
 	}
 	return createStr, nil
 }
+
+// GetTableCollation gets table collation info by 'SHOW TABLE STATUS'.
+// See. https://dev.mysql.com/doc/refman/5.7/en/show-table-status.html
+func GetTableCollation(ctx context.Context, db *sql.DB, schema string, table string) (info string, err error) {
+	querySQL := fmt.Sprintf("SHOW TABLE STATUS from %s like '%s'", schema, table)
```

Comment:
why not using `select table_collation from information_schema.Tables where table_schema="schema" and table_name="table";` ?

this can help reduce scanning logic(L647~L654)

Reply:
yes,  i will change to `information_schema`. I was worried about access `information_schema` rights, but now it seems `show table status` is the same after search mysql reference.
https://dev.mysql.com/doc/refman/8.0/en/information-schema-introduction.html#information-schema-privileges